### DISCUSSION TITLE
为什么明明我把货到付款的页面打开了 但是网页里面只有信用卡付款 请问这是为什么

### DISCUSSION THREAD

**Discussion by 5556548** [woocommerce/woocommerce#50744] [Checkout Flow]
![image](https://github.com/user-attachments/assets/41f49577-7834-411e-9d0b-d9d897b751b4)11111

**Comment by senadir** [comment]
You might have limited your Cash on Delivery to some shipping method? Check the payment method settings.

**Reply by 5556548** [reply]
thank you 我花钱解决了 只需要把地区限制取消就可以解决 感谢你的回答 bro
